### ISSUE TITLE
Apps close exit after start

### ISSUE THREAD

**Issue by jwoertink** [zach-capalbo/flammarion#24]
Just testing this out, I made a file.

```ruby
# app.rb
Flammarion.about
```

Then I ran `ruby app.rb`, and a dialog opened, then closed and the whole app exited with
```text
[23:43PM] 404app3$ ruby app.rb 
(electron) companyName is now a required option to crashReporter.start
(electron) loadUrl is deprecated. Use loadURL instead.
[23:44PM] 404app3$ 
```

I was able to circumvent this by adding a 

```ruby
Flammarion.about
while true
end
```

This seems to be the case for the examples in the README as well.

**Comment by zach-capalbo** [comment]
Thanks for bringing this up! I've changed the `about` method to block (by calling `wait_until_closed`) and updated the Readme to try to make it a little more clear. 

Addressed in [v0.1.5](https://github.com/zach-capalbo/flammarion/releases/tag/v0.1.5)

**Comment by jwoertink** [comment]
Ah, ok. Awesome. Just updated, and I see those updates. Thanks!
